test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -1012,22 +1055,22 @@ def create_kwargs():
         "HF Bloom model does not need `model.load_state_dict` to work."
     )
     def test_fake_tensor_mode_huggingface_bigscience_bloom_560m(self):
-        from transformers import AutoModel, AutoTokenizer  # type: ignore[import]
-
-        model_name = "bigscience/bloom-560m"
+        config = transformers.BloomConfig()
+        batch, seq = 4, 256
 
         def create_args():
             return tuple()
 
-        def create_kwargs(model_name=model_name):
-            tokenizer = AutoTokenizer.from_pretrained(model_name)
-            return tokenizer("Hello world!", return_tensors="pt")
+        def create_kwargs():
```

Comment:
In `_test_fake_tensor_mode_exporter`, the exported ONNX model and PyTorch model are run and their outputs are compared. However, I am wondering if that really checks `dynamic_shapes`. If not, probably with the next (or 2nd next), let's modify `create_args`, `create_kwargs`, and `test_fake_tensor_mode_exporter` to guard `dynamic_shapes` supports different batch and sequence sizes.

Reply:
There is `assert_dynamic_shapes` inside the `_test_fake_tensor_mode_exporter` should address your concern.